src/olmo_core/data/utils.py

```diff
@@ -841,7 +841,7 @@ def pack_documents(
 
         # Sort document indices by document length, decreasing.
         document_lengths = document_indices[:, 1] - document_indices[:, 0]
-        sorted_index = np.argsort(-1 * document_lengths)
+        sorted_index = np.argsort(-1 * document_lengths.astype(np.int64))
```

Comment:
Doing this rather than `sorted_index = np.argsort(document_lengths)[::-1]` preserves the ordering expected by the tests.

Reply:
These tests fail on `main` when I run them on my laptop. Unclear how they are passing when run on github actions.

Reply:
Hmm, seem to be passing for me 🤷